public/index.html

```diff
@@ -21,13 +20,13 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-  <title>nwPlus Live</title>
-</head>
+    <title>HackCamp Portal</title>
```

Comment:
we should add an issue to make this dynamic later (if that's possible)

Reply:
@jackyzha0 pls make issue ty!